File: csharp-netstandard/src/Cloud.Governance.Client/Model/TaskApprovalStatus.cs


using System;
using System.Collections;
using System.Collections.Generic;
using System.Collections.ObjectModel;
using System.Linq;
using System.IO;
using System.Runtime.Serialization;
using System.Text;
using System.Text.RegularExpressions;
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;
using Newtonsoft.Json.Linq;
using System.ComponentModel.DataAnnotations;
using OpenAPIDateConverter = Cloud.Governance.Client.Client.OpenAPIDateConverter;

namespace Cloud.Governance.Client.Model
{
    
    [JsonConverter(typeof(StringEnumConverter))]
    
    public enum TaskApprovalStatus
    {
        [EnumMember(Value = "0")]
        PendingApproval = 0,

        [EnumMember(Value = "1")]
        PendingConfirm = 1,

        [EnumMember(Value = "2")]
        PendingAndCompleted = 2

    }

}

File: typescript/src/models/TaskApprovalStatus.ts
/* tslint:disable */
/* eslint-disable */
/*
 * Cloud Governance Api
 *
 * Contact: support@avepoint.com
 */

/**
 * 
 * @export
 * @enum {string}
 */
export enum TaskApprovalStatus {
    PendingApproval = 0,
    PendingConfirm = 1,
    PendingAndCompleted = 2
}

export function TaskApprovalStatusFromJSON(json: any): TaskApprovalStatus {
    return TaskApprovalStatusFromJSONTyped(json, false);
}

export function TaskApprovalStatusFromJSONTyped(json: any, ignoreDiscriminator: boolean): TaskApprovalStatus {
    return json as TaskApprovalStatus;
}

export function TaskApprovalStatusToJSON(value?: TaskApprovalStatus | null): any {
    return value as any;
}

